documentation/book/con-custom-resources-status.adoc

```diff
@@ -6,15 +6,64 @@
 
 = {ProductName} custom resource status
 
-The `status` property of a {ProductName}-specific custom resource publishes the _current_ state of the resource to users and tools that need the information.
+The `status` property of a {ProductName} custom resource publishes information about the resource to users and tools that need it.
 
-Status information is useful for tracking progress related to a resource achieving its _desired_ state, as defined by the `spec` property. The `status` provides the time and reason the state of the resource changed and details of events preventing or delaying the Operator from realizing the desired state.
+Several resources have a `status` property, as described in the following table.
 
-{ProductName} creates and maintains the status of custom resources, periodically evaluating the current state of the custom resource and updating its status accordingly.
+[cols="3*",options="header",stripes="none",separator=¦]
+|===
 
-When performing an update on a custom resource using `kubectl edit`, for example, its `status` is not editable. Moreover, changing the `status` would not affect the configuration of the Kafka cluster.
+m¦{ProductName} resource
+¦Schema reference
+¦Publishes status information on...
+
+m¦Kafka
+¦xref:type-KafkaStatus-reference[]
+¦The Kafka cluster.
+
+m¦KafkaConnect
+¦xref:type-KafkaConnectStatus-reference[]
+¦The Kafka Connect cluster, if deployed.
+
+m¦KafkaConnectS2I
+¦xref:type-KafkaConnectS2Istatus-reference[]
+¦The Kafka Connect cluster with Source-to-Image support, if deployed.
+
+m¦KafkaMirrorMaker
+¦xref:type-KafkaMirrorMakerStatus-reference[]
+¦The Kafka MirrorMaker tool, if deployed.
+
+m¦KafkaUser
+¦xref:type-KafkaUserStatus-reference[]
+¦Kafka users in your cluster.
+
+m¦KafkaBridge
+¦xref:type-KafkaBridgeStatus-reference[]
+¦The {ProductName} Kafka Bridge, if deployed.
+
+|===
+//When dev work is complete, add `KafkaTopicStatus` to the above table. `KafkaTopicStatus`: Kafka topics and their partitions
+
+The `status` property of a resource provides information on the resource's:
 
-IMPORTANT: The status property feature for {ProductName}-specific custom resources is still under development and only available for Kafka resources.
+* _Current state_, in the `status.conditions` property
+
+* _Last observed generation_, in the `status.observedGeneration` property
+
+The `status` property also provides resource-specific information. For example:
+
+* `KafkaConnectStatus` provides the REST API endpoint for Kafka Connect connectors.
+
+* `KafkaUserStatus` provides the user name of the Kafka user and the `Secret` in which their credentials are stored.
+
+* `KafkaBridgeStatus` provides the HTTP address at which external client applications can access the Bridge service.
```

Comment:
maybe we should say "HTTP based URL" based on the naming we are using in the implementation. Wdyt @tombentley ?

Reply:
Or:
"provides the URL at which external client applications can access the Bridge service"